lib/wallet/src/Cardano/Wallet.hs

```diff
@@ -2593,7 +2593,7 @@ buildAndSignTransactionNew
     -> PreSelection
     -> TransactionCtx
     -> IO (BuiltTx, UTCTime)
-buildAndSignTransactionNew ti db@DBLayer{..} netLayer txLayer pwd walletId
+buildSignSubmitTransaction ti db@DBLayer{..} netLayer txLayer pwd walletId
```

Comment:
In the last commit it seems you added `Submit` to the function name without adding it to the implementation

Reply:
Yes, working on it atm.